biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -930,22 +963,40 @@ subroutine UpdatePlantHydrLenVol(ccohort,site_hydr)
     ! Partition the total absorbing root lengths and volumes into the active soil layers
     ! We have a condition, where we may ignore the first layer
     ! ------------------------------------------------------------------------------
-    
+    ! modified by Junyan May 29, 2020 
+    ! norm = 1._r8 - &
+    !      zeng2001_crootfr(roota, rootb,site_hydr%zi_rhiz(1)-site_hydr%dz_rhiz(1), site_hydr%zi_rhiz(nlevrhiz))
+ 
+   
+    ! set the rooting depth of the cohort, using the logistic functionbelow:   
+    !    z_fr_max/(1 + ((z_fr_max-z_fr_0)/z_fr_0)*exp(-frk*dbh_rev))
+    !  which is constrained by the maximum soil depth:  site_hydr%zi_rhiz(nlevrhiz)
+
+    z_fr = min(site_hydr%zi_rhiz(nlevrhiz), z_fr_max/(1 + ((z_fr_max-z_fr_0)/z_fr_0)*exp(-frk*dbh_rev)))
```

Comment:
This line here is the key component of the PR, right @JunyanDing ? 
It would be good to put your name and date by this line so folks in the future will know the author.

Reply:
you are right @rgknox . Thanks for the suggestion. I will do that.